docs/Formula-Cookbook.md

```diff
@@ -1052,6 +1052,24 @@ The `sockets` method accepts a formatted socket definition as `<type>://<host>:<
 
 Please note that sockets will be accessible on IPv4 and IPv6 addresses by default.
 
+If you only need one socket and you don't care about the name (the default is `listeners`):
+
+```rb
+service do
+  run [opt_bin/"beanstalkd", "test"]
+  sockets "tcp://127.0.0.1:80"
+end
+```
+
+If you need multiple sockets and/or you want to specify the name:
+
+```rb
+service do
+  run [opt_bin/"beanstalkd", "test"]
+  sockets http: "tcp://0.0.0.0:80", https: "tcp://0.0.0.0:443"
+end
+```
```

Comment:
@owenthereal If you're interested in testing this out, all you've got to do is check out this branch locally and then do your usual local install tests.

To check out the branch you have two options.

1. Pure Git solution ([github docs](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/checking-out-pull-requests-locally#modifying-an-inactive-pull-request-locally))
```console
$ cd $(brew --repo)
$ git fetch origin pull/16054/head:revert-16026-support-multiple-sockets-in-service-dsl
From https://github.com/Homebrew/brew
 * [new ref]             refs/pull/16054/head -> revert-16026-support-multiple-sockets-in-service-dsl
$ git switch revert-16026-support-multiple-sockets-in-service-dsl
Switched to branch 'revert-16026-support-multiple-sockets-in-service-dsl'
```

2. Using the GitHub CLI
```console
$ brew install gh
$ cd $(brew --repo)
$ gh pr checkout 16063
```

Then when you're done just switch back to the master branch.

Reply:
I tested it with my homebrew formula, and this works. Thanks!